robottelo/constants/__init__.py

```diff
@@ -330,6 +330,14 @@
     'mirror_complete': "Complete Mirroring",
     'mirror_content_only': "Content Only",
 }
+
+PRODUCT_KEY_RHEL = 'rhel'
```

Comment:
Oh it will be nice to have these and not be repeating these strings throughout the framework/tests.

Reply:
@mshriver I still have Jira card opened to better management of constants to work on. I will make sure we have minimum constants covering all the data.